hermes/bin/src/runtime_extensions/hermes/integration_test/event.rs

```diff
@@ -85,6 +87,10 @@ pub fn execute_event(
 ) -> anyhow::Result<Option<TestResult>> {
     let app_name = HermesAppName("integration-test".to_owned());
 
+    let hermes_home_dir = Cli::hermes_home()?;
```

Comment:
As it is an integration tests, I think we should use `TempDir` for storing `vsf` instances.